engine/polyglot-api/src/main/scala/org/enso/polyglot/runtime/Runtime.scala

```diff
@@ -1665,6 +1673,12 @@ object Runtime {
     final case class DeserializeLibrarySuggestions(libraryName: LibraryName)
         extends ApiRequest
 
+    /** A request to start the background jobs processing. */
+    final case class StartBackgroundProcessing() extends ApiRequest
+
+    /** A notification about started background jobs. */
+    final case class BackgroundJobsStartedNotification() extends ApiNotification
```

Comment:
What is this `BackgroundJobsStartedNotification` good for?

Reply:
It is a notification sent when the engine starts the background jobs processing. It is different from the `StartBackgoundProcessing` which is a command to start the background jobs processing

Reply:
It is a notification sent when the engine starts the background jobs processing. It is different from the `StartBackgoundProcessing` which is a command to start the background jobs processing